<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,4,0,1">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Список городов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tw_city"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Список складов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tw_warehouses"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pb_create_city">
         <property name="text">
          <string>Создать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_update_city">
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_delete_city">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Название</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_name"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
